examples/nodejs/token-vending-machine/README.md

```diff
@@ -62,6 +62,16 @@ MomentoTokenVendingMachine.MomentoTokenVendingMachineApiEndpointE36C2123 = https
 ```
 
 This is the URL of the API Gateway endpoint for your Token Vending Machine. Now you should be able to `curl` this endpoint, and the response will be a temporary Momento API key suitable for use in a browser!
+You should see an output like:
+
+```
+{"authToken":"someShortLivedDisposableToken","expiresAt":1698119397}
+```
+
+You can also pass a tokenId as a query string to your curl command to add context to your token. This can be particularly useful when using [Momento Topics](https://www.gomomento.com/services/topics) as the tokenId can be retrieved from subscription messages, allowing your application to distinguish between different subscribers.
+
+`https://9jkmukxn68.execute-api.us-west-2.amazonaws.com/prod?name=Taylor`
```

Comment:
should we be consistent in the name of the query param and what you say in the docs? you say `tokenId` and then the QP is `name`. kinda seems like the most clear thing to do is just have the QP be named `tokenId` as well. just a nit though, i don't feel too strongly

Reply:
I'll clarify that in this case the `tokenId` is the name. Good call out. The reason I want it to be more relatable to the real world is for customers to be able to derive some sense of the use case